### PR TITLE
added Get Involved component

### LINKED ISSUE #34: Get Involved Component
image
- Import Title and Icon Component (rename `Icon.jsx` to `Social.jsx`)
- Create a data file called `Socials.js` and store it in the data folder (@/data/home/Socials.js)
- Data file contains an array of objects containing: name, icon, link
- Example of mapping: https://github.com/acm-ucr/isa-website/blob/dev/src/components/Footer.jsx

### PR DESCRIPTION
**Pull request by duvalincus**
image

Adds the social sites and centers them. Not sure about the rectanglefill elements. 

Closes #34.

### CODE REVIEW

File: src/components/Involved.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import Social from "@/components/home/Social";
+import { SOCIALS } from "@/data/home/Socials";
+import Title from "@/components/Title";
+
+const Involved = () => {
+  return (
+    <div>
+      <Title text={"Get Involved"} />
```
Comment (seanquiambao) [inline src/components/Involved.jsx]:
```suggestion
      <Title text="Get Involved" />
```

File: src/components/home/Social.jsx [hunk 2]
```diff
@@ -3,13 +3,13 @@ import Link from "next/link";
 const Icon = ({ icon, text, link }) => {
```
Comment (seanquiambao) [inline src/components/home/Social.jsx]: Rename function to social

File: src/data/home/Socials.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  FaDiscord,
+  FaYoutube,
+  FaLink,
+} from "react-icons/fa";
+import { IoMail } from "react-icons/io5";
+
+export const SOCIALS = [
+  {
+    name: "Discord",
+    icon: <FaDiscord className="text-3xl isa-yellow-200" />,
```
Comment (seanquiambao) [inline src/data/home/Socials.js]: for the className, you can create a const javascript string variable called class containing the CSS. This is so we can adjust all styling to all classNames with a single edit, incase we need to make adjustments
```suggestion
    icon: <FaDiscord className={class} />,
```

Comment (seanquiambao) [reply src/data/home/Socials.js]: - icons are too small

File: src/components/Involved.jsx [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+      <div className="flex justify-content-center">
+        {SOCIALS.map((social, index) => {
+          return (
+            <Social
+              key={index}
+              text={social.name}
+              link={social.link}
+              icon={social.icon}
+            ></Social>
```
Comment (seanquiambao) [inline src/components/Involved.jsx]:
```suggestion
            <Social
              key={index}
              text={social.name}
              link={social.link}
              icon={social.icon}
            />
```

File: src/components/home/Social.jsx [hunk 5]
```diff
@@ -3,13 +3,13 @@ import Link from "next/link";
...
   return (
     <Link
-      className="flex flex-col cursor-pointer h-25 w-20 text-isa-yellow-200 items-center"
+      className="flex flex-col cursor-pointer h-25 text-isa-yellow-200 items-center m-2.5"
```
Comment (seanquiambao) [inline src/components/home/Social.jsx]: a little more margining

**Review by seanquiambao — CHANGES_REQUESTED** [review]

**Comment by duvalincus** [comment]
image
latest commit change

File: src/data/home/Socials.js [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+import {
+  FaInstagram,
+  FaFacebookSquare,
+  FaDiscord,
+  FaYoutube,
+  FaLink,
+} from "react-icons/fa";
+import { IoMail } from "react-icons/io5";
+
+const className = "text-5xl isa-yellow-200";
```
Comment (seanquiambao) [inline src/data/home/Socials.js]: try `7xl`

**Review by seanquiambao — CHANGES_REQUESTED** [review]

**Comment by duvalincus** [comment]
image
w/ 7xl

**Review by seanquiambao — APPROVED** [review]

**Pull request merged**